alphadia/constants/default.yaml

```diff
@@ -254,6 +254,8 @@ optimization:
       targeted_update_factor: 1.0
       automatic_update_interval: 0.99
       automatic_update_factor: 1.1
+      maximal_decrease: 0.8
```

Comment:
could you document here what these mean ?

Reply:
I'll add this in a chained PR where I also make this new convergence rule 1) configurable, and 2) available for all parameters.